ADOPTERS.md

```diff
@@ -11,6 +11,9 @@ production users that have added themselves (in alphabetical order):
   The code is Open Source, [see `github.com/chef/automate`](https://github.com/chef/automate).
 - [Scaleway](https://www.scaleway.com/en/) uses the gRPC-Gateway since 2018 to
   serve millions of API requests per day [1].
+- [SpiceDB](https://github.com/authzed/spicedb) uses gRPC-Gateway to handle
```

Comment:
Nit: I prefer to use the definite article 😅. I see it was missed in the `Chef` description so feel free to fix that one too!
```suggestion
- [SpiceDB](https://github.com/authzed/spicedb) uses the gRPC-Gateway to handle
```